WordPress/Sniffs/PHP/StrictInArraySniff.php

```diff
@@ -101,19 +101,4 @@ public function process_parameters( $stackPtr, $group_name, $matched_content, $p
 		}
 	}
 
-	/**
-	 * Process the function if no parameters were found.
-	 *
-	 * @since 0.11.0
-	 *
-	 * @param int    $stackPtr        The position of the current token in the stack.
-	 * @param array  $group_name      The name of the group which was matched.
-	 * @param string $matched_content The token content (function name) which was matched.
-	 *
-	 * @return void
-	 */
-	public function process_no_parameters( $stackPtr, $group_name, $matched_content ) {
```

Comment:
What value was this providing before, that it no longer needed?

Reply:
None really ;-) I think I added it because we could, not so much because it added real value.

We're not checking if other PHP/WP functions which expect parameters receive them and PHP will throw a warning when it receives too few parameters anyhow.

Aside from that, this check widens the scope of the sniff to more than its mandate.